app/views/layouts/application.slim

```diff
@@ -7,14 +7,17 @@ html
     = stylesheet_link_tag 'application', media: 'all'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
   body
+    p
+      = link_to 'new user by devise', new_user_registration_path
     #notice
     = link_to 'User', users_path
     br
     = link_to 'Apartments', apartments_path
-    - if current_user.present?
-      = link_to "Logout, #{current_user.full_name}", destroy_user_session_path, method: :delete
-    -else
-      = link_to 'Login', new_user_session_path
-      = link_to 'Sign_up', new_user_registration_path
+    div
```

Comment:
why do you use `div` ? If you use div, it's common to add css class to this `div` and slim syntax wil be `.css_class`. And I've already add navigation https://github.com/DenisUA/Ruby-hotel/commit/13310cbd31dc99ed7e007cd224a089b359f75857 , so due to resolve future conflicts please revert changes in application.slim